dubbo-rpc/dubbo-rpc-dubbo/src/main/java/org/apache/dubbo/rpc/protocol/dubbo/DubboInvoker.java

```diff
@@ -107,8 +107,6 @@ protected Result doInvoke(final Invocation invocation) throws Throwable {
 
             invocation.setAttachment(TIMEOUT_KEY, String.valueOf(timeout));
 
-            RpcContext.getServiceContext().setRemoteAddress(currentClient.getRemoteAddress());
```

Comment:
![image](https://github.com/apache/dubbo/assets/9292748/d9de2170-29e6-4644-94cb-894e491214e7)

Check style should be fixed.

Reply:
thanks,I will fix